_labo-help/similarity.markdown

```diff
@@ -24,52 +26,51 @@ The Similarity tool offers the following features and functionality.
 
 ![Similarity tool regions](/uploads/similarity-tool2.jpg)
 
-### <a name="keyframes"></a>Keyframes
-
-Keyframes are shown in a grid under the media viewer. It provides a visual and auditory overview of the current video.
-
-Keyframes have been detected using the scene detect library. They are accompanied by spectrograms of 1 second audio centered around the keyframe.
+### <a name="keyframes"></a>Current resource keyframes
 
-When the video is playing, the currently active keyframe is highlighted. Clicking on a tile moves the player video position to the keyframe timestamp. This way one can navigate the video, in addition to the left/right buttons that move playback to the previous or next keyframe.
+Keyframes have been selected using the [scene detect library](https://www.scenedetect.com/). They are accompanied by spectrograms of 1 second audio centered around the keyframe.
 
-Using the annotation button, annotations can be made on a segment that spans the keyframe start and end time.
+The keyframes are shown in a grid under the media viewer. They provide a visual and auditory overview of the current video. When the video is playing, the currently active keyframe is highlighted. Clicking on a tile moves the player video position to the keyframe timestamp. This way one can navigate the video, in addition to the left/right buttons that move playback to the previous or next keyframe.
 
-The keyframes grid is also available in the resource viewer. It provides the same control about the media playback as in the similarity tool.
+The keyframes grid is also available in the resource viewer. It provides the same control for the media playback as in the similarity tool.
 
 ### <a name="related-keyframes"></a>Related keyframes
 
-Related keyframes are loaded for the active keyframe. Results can be controlled using the query options. Use the info icon behind each input to learn more about its use!
+Related keyframes are loaded for the active keyframe. Results can be controlled using the query options. Use the info icon behind each input to learn more about its use. 
```

Comment:
can be controlled using the query options -> can be controlled using the options available in the taskbar above the related keyframes.

(I think people may misunderstand 'query options' as being like the controls in the Search tool)